doc/dao/Getting Started with Development Environment & Running Primero.md

```diff
@@ -0,0 +1,291 @@
+# Development Environment
```

Comment:
I would like to consolidate these developer instructions with what's in the top-level README. Right now this is partly duplicative. I am OK with consolidating these two guides into one (and I like the animations here!). Maybe a single file `doc/GettingStarted.md` that's referenced both by the top-level README and this one?

Reply:
I've consolidated the documentation somewhat and moved the getting started doc up one directory level. I've put a link in the main readme to the newly moved getting started with development doc. Let me know if there's anything you think is missing from the new doc.